### PR TITLE
Add more fields for load expert filters.

### PR DESCRIPTION
**Pull request by AAJELLAL**

### CODE REVIEW

**Review by YenguiSeddik — APPROVED** [review]
code & test: ok

File: src/translations/en.json [hunk 1]
```diff
@@ -266,5 +266,7 @@
   "downloadCases": "Download Case(s)",
 
   "descriptionModificationError": "An error when modifying the description",
-  "descriptionLimitError": "Description exceeds character limit"
+  "descriptionLimitError": "Description exceeds character limit",
+  "p0": "Constant P",
```
Comment (thangqp) [inline src/translations/en.json]: Could you add two keys after "vlId": "Voltage level ID"  for grouping fields supported by expert filter?

File: src/translations/fr.json [hunk 2]
```diff
@@ -281,5 +281,7 @@
   "downloadCases": "Télécharger la(les) situation(s)",
 
   "descriptionModificationError": "Erreur lors de la modification de la description",
-  "descriptionLimitError": "La description dépasse la limite de caractères"
+  "descriptionLimitError": "La description dépasse la limite de caractères",
+  "p0": "P constant",
```
Comment (thangqp) [inline src/translations/fr.json]: idem, could you add two keys after "vlId": "Voltage level ID"  for grouping fields supported by expert filter?

File: src/translations/en.json (hunk 1 above)
Comment (AAJELLAL) [reply src/translations/en.json]: Done.

File: src/translations/fr.json (hunk 2 above)
Comment (AAJELLAL) [reply src/translations/fr.json]: Done.

**Review by thangqp — APPROVED** [review]

**Pull request merged**